docker/install.sls

```diff
@@ -36,10 +38,11 @@ docker-package:
         {%- endif %}
         {%- if grains.os_family in ('Suse',) %}   ##workaround https://github.com/saltstack-formulas/docker-formula/issues/198
   cmd.run:
-    - name: /usr/bin/pip install {{ '--upgrade' if docker.pip.upgrade else '' }} pip
+    - name: /usr/bin/pip{{ python:major }}.{{ python_minor }} install {{ '--upgrade' if docker.pip.upgrade else '' }} pip
```

Comment:
Appears to be a typo: `{{ python:major }}` => `{{ python_major }}`.

Reply:
```suggestion
    - name: /usr/bin/pip{{ python_major }}.{{ python_minor }} install {{ '--upgrade' if docker.pip.upgrade else '' }} pip
```